### PR TITLE
Map 141 documentation update

### PR DESCRIPTION
**Pull request by jwillmartin**
# PR Details
## Description

These changes are meant to help users install the MAP Tool locally or using docker and with/without SSL certificates. It includes documentation with steps to take and example dockerfiles. 

## Related Jira Key

MAP-141

## Motivation and Context

Users want to install the tool using both methods previously listed. We need documentation for users to use to achieve this. 

## How Has This Been Tested?

Documentation followed to install both locally and using docker. 

## Types of changes

- [ ] Defect fix (non-breaking change that fixes an issue)
- [X] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that cause existing functionality to change)

## Checklist:

- [ ] I have added any new packages to the sonar-scanner.properties file
- [X] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [X] I have read the [**CONTRIBUTING**](https://github.com/usdot-fhwa-stol/carma-platform/blob/develop/Contributing.md) document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/BingMaps_API_Key_Guidance.md [hunk 1]
```diff
@@ -11,5 +11,4 @@ An account with Bing Maps can be created using the [Bing Maps Portal](https://ww
 
 ### **Using your API Key**
 
-#### To do: Create Local and Docker build instructions.
-Once a key is obtained, please insert your key and any arbitrary username in the [ISDcreator-webapp-keys](/private-resources/js/ISDcreator-webapp-keys.js) file and build your updated ConnectedVCS Tools using either the [Local](/docs/) or [Docker](/docs/) build instructions.
+Once a key is obtained, please insert your key and any arbitrary username in the [ISDcreator-webapp-keys](/private-resources/js/ISDcreator-webapp-keys.js) file and build your updated ConnectedVCS Tools using either the [Local](/docs/Local_Instructions.md) or [Docker](/docs/Docker_Instructions.md) build instructions.
```
Comment (DokurOmkar) [inline docs/BingMaps_API_Key_Guidance.md:14]: It seems like Bing Map API keys are deprecated and new users will not be able to create these keys.

image

We may have to add a note here that new keys will be deprecated.

File: docs/Docker_Instructions.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+## Run the ConnectedVCS Tools Image
+
+### Build a Custom Image
+
+1. Clone the ConnectedVCS Tools respository:
+```
+git clone https://github.com/usdot-fhwa-stol/connectedvcs-tools.git
+```
+2. Create a new Bing Maps API Key (Basic, Website) using https://www.bingmapsportal.com/.
```
Comment (DokurOmkar) [inline docs/Docker_Instructions.md:19]: It seems like Bing Map API keys are deprecated and new users will not be able to create these keys. Please see the above comment.

File: docs/Local_Instructions.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+1. Set up LD_LIBRARY_PATH by running:
+```
+echo export LD_LIBRARY_PATH="[path_to_connectedvcs-tools]/fedgov-cv-lib-asn1c/third_party_lib" >> ~/.bashrc
+```
+   - **NOTE**: Change the above path to point to the correct [third_party_lib path](/fedgov-cv-lib-asn1c/third_party_lib).
+
+## Local Build Instructions
+
+1. Create a new Bing Maps API Key (Basic, Website) using https://www.bingmapsportal.com/.
```
Comment (DokurOmkar) [inline docs/Local_Instructions.md:33]: Same here. Add that it is deprecated.

File: private-resources/WEB-INF/lib/cas-client-core-3.1.10.jar
Comment (DokurOmkar) [inline private-resources/WEB-INF/lib/cas-client-core-3.1.10.jar:1]: Do we need these jar files?

File: private-resources/index.html [hunk 4]
```diff
@@ -0,0 +1,6 @@
+<!DOCTYPE html>
```
Comment (DokurOmkar) [inline private-resources/index.html:1]: Do we need this empty index.html?

File: private-resources/META-INF/maven/gov.usdot/fedgov-cv-private-resources-webapp/pom.xml [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+		<version>1.0.0-SNAPSHOT</version>
+		<relativePath>../fedgov-cv-parent/pom.xml</relativePath>
+	</parent>
+	<properties>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+		<java.version>1.8</java.version>
+	</properties>
+	<dependencies>
+		<dependency>
+			<groupId>org.jasig.cas</groupId>
```
Comment (DokurOmkar) [inline private-resources/META-INF/maven/gov.usdot/fedgov-cv-private-resources-webapp/pom.xml:20]: Is this dependency required?

**Review by DokurOmkar — CHANGES_REQUESTED** [review]
Bing Maps seems to be deprecated lately that requires us to change the documentation.

Also, do we need to pass the keys from private resources? If yes, do we need those jar files/dependencies in pom.xml file?

File: docs/BingMaps_API_Key_Guidance.md (hunk 1 above)
Comment (jwillmartin) [reply docs/BingMaps_API_Key_Guidance.md:14]: Note added. New story will be created to migrate from Bing Maps.

File: private-resources/WEB-INF/lib/cas-client-core-3.1.10.jar
Comment (jwillmartin) [reply private-resources/WEB-INF/lib/cas-client-core-3.1.10.jar:1]: Yes. I tried to remove them a while back and it broke functionality

File: private-resources/index.html (hunk 4 above)
Comment (jwillmartin) [reply private-resources/index.html:1]: Not needed, removed

File: private-resources/META-INF/maven/gov.usdot/fedgov-cv-private-resources-webapp/pom.xml (hunk 5 above)
Comment (jwillmartin) [reply private-resources/META-INF/maven/gov.usdot/fedgov-cv-private-resources-webapp/pom.xml:20]: Yes, functionality was broken when it was removed

**Review by DokurOmkar — APPROVED** [review]

**Pull request merged**